### ISSUE TITLE
Outer most 2 braces are not matched in a file with more than 401 lines

### ISSUE THREAD

**Issue by Hans-Guenter** [andymass/vim-matchup#21]
The outer most 2 braces are not matched in the attached file using % from matchup
If one joins 2 lines (thus reducing the number of lines by 1) they match.

Another finding is that if one places the cursor on line 2 or on the penultimate line of this file there is a weird entry on the statusline.

In the case of the file with 2 lines joined this weird status line shows up if the cursor is on line 1, 2, the last or the penultimate line.
[Matchup_Problem_File.txt](https://github.com/andymass/vim-matchup/files/1713229/Matchup_Problem_File.txt)

**Comment by andymass** [comment]
> the outer most 2 braces are not matched in the attached

The number of search lines is limited to 400 for performance reasons.  The recent commit upgrades this to 1500 for motions like %.  There is a separate stopline setting for highlighting and motions.  See the options `g:matchup_delim_stopline` and `g:matchup_matchparen_stopline` for configuration, which are now documented.

> there is a weird entry on the statusline

This is a feature to show you matches which are off of the screen.  However, I corrected a bug which was showing it globally instead of locally.  To disable the feature `let g:matchup_matchparen_status_offscreen = 0`.  Updated documentation to make that clear.
